packages/devextreme/testing/testcafe/tests/dataGrid/columnHeaders.ts

```diff
@@ -0,0 +1,19 @@
+import { a11yCheck } from '../../helpers/accessibilityUtils';
+import url from '../../helpers/getPageUrl';
+import createWidget from '../../helpers/createWidget';
+
+fixture.disablePageReloads`Column Headers`
+  .page(url(__dirname, '../container.html'));
+
+test('Checking column headers via aXe', async (t) => {
+  await a11yCheck(t);
```

Comment:
You need to ensure the method perform contrast checks. Now they are excluded.

Reply:
I suggest to add test for material theme.